Meshery-Adapters/linkerd-meshery-adapter/step3.md

```diff
@@ -46,12 +46,13 @@ Sample Output:
 |   web    |  1/1  |      1     |     1     | 4m38s |
 
 ### Let's inspect our pod:
+If you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When all the deployment are in ready state, you can use `CTRL+ C` to switch back to the terminal.
 
 1. View the details of the deployed pods:
 
     `watch kubectl get po -n default`{{execute}} 
 
-If you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When the deployment is complete, the page will auto-refresh. You can use `CTRL+ C` to switch back to the terminal.
+Again if you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When the deployment is complete, the page will auto-refresh. You can use `CTRL+ C` to switch back to the terminal.
```

Comment:
```suggestion
Again if you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When the deployment is complete, the page will auto-refresh. You can use `CTRL+ C` or `CMD + C` to switch back to the terminal.
```